src/Functions/randomString.cpp

```diff
@@ -44,9 +44,8 @@ class FunctionRandomString : public IFunction
                 "Function " + getName() + " requires at most two arguments: the size of resulting string and optional disambiguation tag",
                 ErrorCodes::NUMBER_OF_ARGUMENTS_DOESNT_MATCH);
 
-        const IDataType & length_type = *arguments[0];
-        if (!isNumber(length_type))
-            throw Exception("First argument of function " + getName() + " must have numeric type", ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT);
+        if (!isUnsignedInteger(*arguments[0]))
```

Comment:
This is unnecessary change,
consider `randomString(length - 1)`

length - 1 is having signed type even if length is unsigned.